pilot/pkg/serviceregistry/kube/conversion.go

```diff
@@ -112,6 +112,8 @@ func convertService(svc v1.Service, domainSuffix string) *model.Service {
 		LoadBalancingDisabled: loadBalancingDisabled,
 		MeshExternal:          meshExternal,
 		Resolution:            resolution,
+		ID:                    fmt.Sprintf("%s:%s", svc.Namespace, svc.Name),
```

Comment:
name,namespace .. not other way around?

Reply:
Nate's PR will take care of this - will remove from my PR.